lnwallet/chainfee/estimator.go

```diff
@@ -816,22 +816,24 @@ func (w *WebAPIEstimator) EstimateFeePerKW(numBlocks uint32) (
 //
 // NOTE: This method is part of the Estimator interface.
 func (w *WebAPIEstimator) Start() error {
+	log.Infof("Starting Web API fee estimator...")
```

Comment:
yeah you are totally right! For some reason, I thought it was waiting for the ticker to fire before doing the first `updateFeeEstimate`. Will remove this commit. Meanwhile I realized that if `Start` is still finishing up, yet another call to `EstimateFeePerKW` is made, we could still get a floor feerate. This should never happen as we always wait for the `Start` to be finished first, but just in case.

Reply:
This can be avoided by inlining the Start method into the constructor.